sdk/keyvault/keyvault-secrets/src/secretsModels.ts

```diff
@@ -87,13 +87,13 @@ export interface SecretProperties extends ParsedKeyVaultEntityIdentifier {
    * **NOTE: This property will not be serialized. It can only be populated by
    * the server.**
    */
-  readonly created?: Date;
```

Comment:
Does removing the `readonly` keyword fix something?

Reply:
I'll try to put them back. I needed to set these properties if they existed in another object. The conversions get kind of annoying